src/master_element/MasterElementHO.C

```diff
@@ -1776,7 +1776,7 @@ HigherOrderEdge2DSCS::HigherOrderEdge2DSCS(
   basis_(basis),
   quadrature_(quadrature)
 {
-  nDim_ = 2;
+  ndim(2);
```

Comment:
AlgTraits::nDim_ instead of '2'?

Reply:
The higher order elements do not have an AlgTraits associated with it.  No fixed integration order for instance.